STYLE_GUIDE.md

```diff
@@ -33,7 +33,7 @@
 - Non-local Goto
   - There should be no calls to the functions `setjmp` or `longjmp`.
 - Local Goto
-  - There should be no calls to the C/C++ keyword goto.  Exception: The use of local gotos for the purposes of common error handling blocks and single points of function return at the bottom of a function.
+  - The use of C/C++ keyword goto should be avoided.  Exception: The use of local gotos for the purposes of common error handling blocks and single points of function return at the bottom of a function.
```

Comment:
I’m not sure this change makes a meaningful difference. The original text already has “should” rather than “shall”, so the original text already allows exceptions. Thoughts?

Reply:
Sounds good to me ... as long as we can use `goto` seletively where it makes sense... :)
Will close this.